pkg/minikube/openshiftversions/openshift_versions.go

```diff
@@ -17,48 +17,119 @@ limitations under the License.
 package openshiftversions
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
-
```

Comment:
I removed github stuff completely and using docker registry for upstream and access.registry.redhat.com for downstream.